src/domain/platform/admin/subspace/SubspaceSettingsLayout.tsx

```diff
@@ -52,20 +52,19 @@ const SubspaceSettingsLayout: FC<SubspaceSettingsLayoutProps> = props => {
     ];
 
     if (journeyLevel === 1) {
-      tabs.push(
-        {
-          section: SettingsSection.Subsubspaces,
-          route: 'opportunities',
-          icon: FlagOutlinedIcon,
-        },
-        {
-          section: SettingsSection.SpaceSettings,
-          route: 'settings',
-          icon: GppGoodOutlinedIcon,
-        }
-      );
+      tabs.push({
+        section: SettingsSection.Subsubspaces,
+        route: 'opportunities',
+        icon: FlagOutlinedIcon,
+      });
     }
 
+    tabs.push({
```

Comment:
Potentially the page will be empty for opportunity admins (non-creators). We can think of default text there, e.g. "No additional settings available on that level."